sale_restricted_qty/__manifest__.py

```diff
@@ -1,15 +1,16 @@
+# Update (Migration) 2022 Ooops - Ashish Hirpara
 # Copyright 2019 Akretion (<http://www.akretion.com>)
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
 {
     "name": "Sale order min quantity",
-    "version": "12.0.2.0.0",
+    "version": "14.0.1.0.0",
     "category": "Sales Management",
-    "author": "Akretion, " "Odoo Community Association (OCA)",
-    "website": "https://github.com/OCA/sale-workflow/",
+    "author": "Ashish Hirpara, Akretion, Odoo Community Association (OCA)",
```

Comment:
@AshishHirapara Migration is not valid for changing authorship. Please add yourself to CONTRIBUTORS instead (same for headers).

Reply:
ok, I will update the changes. Thanks for the info.

Reply:
> @AshishHirapara Migration is not valid for changing authorship. Please add yourself to CONTRIBUTORS instead (same for headers).

@rousseldenis  This is updated.